docs/build.md

```diff
@@ -2,40 +2,88 @@
 
 Dependency | Version | Required           | Runtime
 ---        | ---     | ---                | ---
-Kokkos     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
+[Kokkos](https://github.com/kokkos/kokkos)     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
 MPI        | 2       |                    | :heavy_check_mark:
 CMake      | 3.12    | :heavy_check_mark: |
```

Comment:
No link for `CMake`?

Reply:
No. I think that people may not know what Kokkos is, but if they don't know CMake, we can't really help it.